tests/modules/custom-status/rest/test-custom-status-endpoint.php

```diff
@@ -17,15 +18,18 @@
 class CustomStatusRestApiTest extends RestTestCase {
 
 	/**
-	 * Before each test, ensure default custom statuses are available.
+	 * Before each test, ensure default custom statuses are available and reset all module options.
 	 */
 	protected function setUp(): void {
 		parent::setUp();
 
+		// Reset all module options
+		OptionsUtilities::reset_all_module_options();
```

Comment:
If we need to reset options and install for each test, we could put these in `WorkflowTestCase` `setUp()`/`tearDown()` methods.

Reply:
Addressed in [this comment below](https://github.com/Automattic/vip-workflow-plugin/pull/59#issuecomment-2402886191).